sdk/src/test/scala/io/horizen/account/block/AccountBlockHeaderTest.scala

```diff
@@ -64,7 +70,7 @@ class AccountBlockHeaderTest extends JUnitSuite with CompanionsFixture with Acco
     assertEquals("AccountBlockHeader id is different", header.id, serializedHeader.id)
 
     // Set to true to regenerate regression data
-    if(false) {
+    if(true) {
```

Comment:
Updating was necessary because the test data had a timestamp in the block that was older than the genesis block, which went unnoticed so far but caused issues now. Comitting the `true` here was a mistake though, I will remove this.